### PR TITLE
Popup task completed

### PR DESCRIPTION
**Pull request by marynatovtyn**
### hooli-popup

[Demo](https://marynatovtyn.github.io/hooli-style-popup/) |
[Code base](https://github.com/marynatovtyn/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by marynatovtyn** [comment]
Completed self-checks, ready for review

File: submissions/marynatovtyn/hooli-style-popup/style.css [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+.checkbox-more:checked ~ .more__list {
+    display: grid;
+}
+
+.checkbox-more:checked ~ .checkbox-more__label {
+    display: none;
+}
+
+.more__list {
+    /* display: grid; */
```
Comment (A-Ostrovnyy) [inline submissions/marynatovtyn/hooli-style-popup/style.css:209]: Avoid commented code in your work.

File: submissions/marynatovtyn/hooli-style-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    display: none;
+    grid-template: repeat(2, 1fr) / repeat(3, 1fr);
+    gap: 10px;
+    padding: 10px;
+    justify-items: center;
+    align-items: center;
+}
+
+::-webkit-scrollbar {
```
Comment (A-Ostrovnyy) [inline submissions/marynatovtyn/hooli-style-popup/style.css:218]: This is an invalid selector: you can't set pseudoelement from an empty selector.

File: submissions/marynatovtyn/hooli-style-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    border-radius: 5px;
+}
+
+::-webkit-scrollbar-thumb {
+    background-color: #9c9195;
+    border-radius: 5px;
+}
+
+.main {
+    background-color: #EEE2DC;
```
Comment (A-Ostrovnyy) [inline submissions/marynatovtyn/hooli-style-popup/style.css:233]: You can add this style to the `body` and remove this element from markup)

File: submissions/marynatovtyn/hooli-style-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+}
+
+.header__link {
+    display: block;
+    font-size: 20px;
+    padding: 5px;
+}
+
+.header__link:focus {
+    outline: 0;
```
Comment (A-Ostrovnyy) [inline submissions/marynatovtyn/hooli-style-popup/style.css:67]: better use `outline: none`

File: submissions/marynatovtyn/hooli-style-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+.header__menu {
+    grid-column: 2 / 3;
+    width: 100%;
+}
+
+.header__list {
+	display: flex;
+    justify-content: end; 
+    align-items: center;
+    position: relative;
```
Comment (A-Ostrovnyy) [inline submissions/marynatovtyn/hooli-style-popup/style.css:40]: Please fix indentation

File: submissions/marynatovtyn/hooli-style-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+              </a>
+            </li>
+            <li class="header__item">
+              <a href="#" class="header__link">
+                <img
+                  src="./img/cat.png"
+                  alt="Profile picture"
+                  class="header__profile"
+                />
+              </a>
```
Comment (A-Ostrovnyy) [inline submissions/marynatovtyn/hooli-style-popup/index.html:169]: Add styles on hover and focus state.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome work!

**Comment by marynatovtyn** [comment]
Thanks for your review! I added requested changes

File: submissions/marynatovtyn/hooli-style-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main"></main>
```
Comment (A-Ostrovnyy) [inline submissions/marynatovtyn/hooli-style-popup/index.html:174]: Avoid unused markup in your work)

**Review by A-Ostrovnyy — APPROVED** [review]

**Pull request merged**